source/development_process/index.rst

```diff
@@ -0,0 +1,179 @@
+all developers should the team choose to follow this methodology. When
+following the agile methodology the developer team meets every month to
+discuss the state of the project and show their progress to the client. The Sea
+Grant projects also followed the test driven development model, which dictates
+that tests should be written before code. This ensures that all of the code is
+tested and works properly.
+
+The PGD project has a very different set of constraints. It is a large project
+which is being maintained, but little active development is being done. The PGD
+is a tool used by hundreds of scientists throughout the world, so it is very
+important that changes do not introduce bugs which may affect ongoing research.
+Every change to the PGD project must be approved by the scientists in charge of
+the project. It is difficult to write tests to ensure that new changes don't
+impact the science. Historically the PGD has had issues with upstream protein
+repositories and antiquated closed-source dependencies. Even importing new data
+can sometimes be a fraught process, but we are developing a detailed process to
+minimize problems when upstream software or data changes.
+
+Tools We Use
+------------
+At the OSL we use a variety of open source tools. Developers are expected to
+have at least passing familiarity with all of these tools. New developers are
+encouraged to research any of these tools they have not heard of.
+
+Most OSL websites are written in Python using the Django web application
+framework. Websites are deployed using Chef configuration management on CentOS
+6.6. Most of our sites use the MySQL database, although some use PostGres.
+Most OSL websites have a Docker development environment set up to make
+development fast and easy. To learn how to use the Docker development
+environment please consult the project documentation and the `OSL's internal
+documentation
+<https://docs.osuosl.org/development/docker-dev-environments.html>`_.
+The OSL uses ``git`` for version control and keeps projects on
+`GitHub <https://github.com/osuosl>`_. Some legacy
+projects such as Ganeti Web Manager use our `internal gitolite server
+<git.osuosl.org>`_. The OSL uses the `RedMine issue tracker
+<https://code.osuosl.org>`_ because GitHub does not support time tracking.
+
+How a Typical Project is Written
+--------------------------------
+A step by step guide to writing the project
+
+Determine Requirements
+----------------------
+
+- Meet with client
+- determine requirements
+- determine milestones
+- internally determine code sprints
+
+Setting Up the Project
+----------------------
+
+- Create the git repo
+- Create the issue tracker
+- Create the project skeleton with Django
+- Create the Dockerfile
+
+The Development Process
+-----------------------
+
+- Pick an issue
+- Write tests (with a sample Django test)
+- Write the code
+- Make a PR
+- Merge it
+- Close the issue
+
+A Brief Introduction to Django
```

Comment:
I'm beginning work on this section. Should this include things like "making a virtualenv/installing Big Project's pip requirements", or start with the assumption that you have a venv set up and django installed?